pkgs/development/compilers/openspin/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "openspin-${version}";
+  version = "2018-02-26";
+
+  src = fetchFromGitHub {
+    owner = "parallaxinc";
+    repo = "OpenSpin";
+    rev = "7bbc030c2967e5d266a076c0a1a5ce4c21ae7b06";
+    sha256 = "19xxfapvyskimpkg9g76rd9j4gcfikmpdx2mf599w1yzwcnx10s9";
+  };
+
+  doCheck = true;
+
+  postPatch = ''
+    sed -i -e '1,18d' -e '20d' -e 's/  BUILD=/BUILD=/' Makefile
+    sed -i -e '1,18d' -e '20d' -e 's/  BUILD=/BUILD=/' PropellerCompiler/Makefile
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mv build/openspin $out/bin/openspin
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Compiler for SPIN/PASM languages for Parallax Propeller MCU";
+    longDescription = ''
+      Compiler for SPIN/PASM languages for Parallax Propeller MCU.
```

Comment:
No need to repeat the `description` in the `longDescription`.

Reply:
Should I just leave the long description blank?

Reply:
No need to have `longDescription`.